tests/bpftrace.cpp

```diff
@@ -1441,9 +1441,8 @@ TEST(bpftrace, print_basic_map)
     std::string expected_output;
   };
 
-  auto keys = std::vector<uint64_t>{ 1, 3, 5, 7, 9 };
-  auto values = std::vector<uint64_t>{ 5, 10, 4, 11, 7 };
-  auto returned_kvs = generate_kv_pairs(keys, values);
+  const auto keys = std::vector<uint64_t>{ 1, 3, 5, 7, 9 };
+  const auto values = std::vector<uint64_t>{ 5, 10, 4, 11, 7 };
```

Comment:
Nit: changes in this file look unrelated to the rest. Maybe move them to a separate commit?